### ISSUE TITLE
add archivability as a best practice for .gov website design/development

### ISSUE THREAD

**Issue by nullhandle** [18F/web-design-standards#734]
Great work! The U.S. Web Design Standards looks like a great resource. I wonder if you might also consider adding [guidance on building websites so as to facilitate archiving](https://library.stanford.edu/projects/web-archiving/archivability)?

The ability to effectively access and use government information depends not just on accessibility and usability but also on persistence, and U.S. Federal Government websites are no exception to the general characteristic of web content to change or disappear. In addition to the [web-wide archiving efforts of the Internet Archive](https://archive.org/about/faqs.php#The_Wayback_Machine), there are more targeted efforts focused specifically on U.S. Federal Government websites, including the [Federal Web Archiving Working Group](http://blogs.loc.gov/digitalpreservation/2015/02/introducing-the-federal-web-archiving-working-group/); the [University of North Texas CyberCemetery](http://govinfo.library.unt.edu/); the [2008](http://eotarchive.cdlib.org/background.html), [2012](http://eotarchive.cdlib.org/2012.html), and forthcoming 2016 End of Term Web Archive projects; and [many other distributed efforts](https://archive-it.org/explore?show=Collections&fc=meta_Subject%3AGovernment-usfederal).

The success of all of these efforts depends on the amenability of the websites to archiving. The U.S. Web Design Standards could have a major positive impact in this regard. Please let me know if I can help make this happen. Thanks!

**Comment by mollieru** [comment]
Very cool suggestion, thank you! Can you tell us a little bit more about what kind of language/information you'd like to see here in order to encourage archiving?

**Comment by nullhandle** [comment]
Thanks for your interest. There are many recommendations in [the documentation](https://library.stanford.edu/projects/web-archiving/archivability) I linked to above. Most of those are appropriate to the U.S. Federal Government web context.

If I had to distill down to a more limited and prioritized set of recommendations, they'd be:

* Make links discoverable; make links meaningful (see sections "Make links transparent" and "Represent web app states with links" on the [Capture](https://library.stanford.edu/projects/web-archiving/archivability/capture) page)
* Make good use of robots directives (see section "Be careful with robots directives" on the [Capture](https://library.stanford.edu/projects/web-archiving/archivability/capture) page)
* Return reliable and useful HTTP responses (see sections "Return reliable response codes" and "Implement caching enhancements" on the [Capture](https://library.stanford.edu/projects/web-archiving/archivability/capture) page)

I've invited the web archivists involved in the aforementioned initiatives to weigh in here, as well; they can offer additional perspective based on their own experiences attempting to archive U.S. Federal Government websites. I suspect they might also be willing to draft more specific language and guidelines for co-inclusion with the U.S. Web Design Standards, depending on what you're amenable to or how you see it potentially fitting in.
